src/components/Restoration/RestorationType.vue

```diff
@@ -3,38 +3,45 @@
     <v-row>
 
       <v-col cols="12" sm="3" class="pr-4">
-        <label class="font-weight-bold">Restoration Type</label>
+        <label
+        v-bind:class="{'invalid': !getRestorationTypeValid, 'valid-title': getRestorationTypeValid}"
```

Comment:
You can leave out `v-bind`.

You probably don't need exclusive conditional classes -- set one class here and then let CSS specificity overwrite it conditionally.